<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CcMusic</class>
 <widget class="QWidget" name="CcMusic">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="2,6,3">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="WgtTop" native="true"/>
   </item>
   <item>
    <widget class="QWidget" name="WgtMain" native="true"/>
   </item>
   <item>
    <widget class="QWidget" name="WgtBottom" native="true"/>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
